File: softuni-js-basics/01-first-steps-in-coding/exercises-submitted-evaluated/09.yardGreening.js
function yardGreening(input) {
    let pricePerSquareMeter = 7.61;
    let discont = (18 / 100);
    let squareMeters = Number(input[0]);
    let finalPriceBeforeDiscount = squareMeters * pricePerSquareMeter;
    let discountValue = finalPriceBeforeDiscount * discont;
    let finalPriceAfterDiscount = finalPriceBeforeDiscount - discountValue;

    console.log(`The final price is: ${finalPriceAfterDiscount} lv.`);
    console.log(`The discount is: ${discountValue} lv.`);
}

yardGreening(["550"]);

File: softuni-python-basics/01_first_steps_in_coding/09_yard_greening.py
sqm_for_greening = float(input())
greening_price = sqm_for_greening * 7.61
discount = 0.18 * greening_price
greening_price -= discount
print(f"The final price is: {greening_price} lv.")
print(f"The discount is: {discount} lv.")
